packages/projection-typeorm/src/entity/CurrentPoolMetrics.entity.ts

```diff
@@ -11,37 +11,40 @@ export class CurrentPoolMetricsEntity {
   @PrimaryColumn({ length: 56, type: 'char' })
   stakePoolId?: Cardano.PoolId;
 
-  @Column()
+  @Column({ nullable: true })
   slot: Cardano.Slot;
 
   @OneToOne(() => StakePoolEntity, OnDeleteCascadeRelationOptions)
   @JoinColumn()
   stakePool?: StakePoolEntity;
 
-  @Column({ type: 'integer' })
+  @Column({ nullable: true, type: 'integer' })
```

Comment:
We should probably have a cardano-services migration too in this commit

Reply:
Yes sure!

I'll add migrations once all the changes on schema will be reviewed / approved to add them only once rather than applying them now with the risk of need to change them during the review process (with the relative higher developer time cost).

Reply:
1b2570abe should solve this

Reply:
1b2570abe should solve this

Reply:
Probably doesn't matter for this table, but `RENAME COLUMN column_name TO new_column_name;` would preserve any existing values